architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,194 @@
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Authorization policy determines whether a user is allowed to perform a given
+link:#action[action]. This allows platform administrators to use
+link:#cluster-policy-and-local-policy[cluster policy] to control who has various
+access levels to the overall OpenShift platform itself. It also allows
+developers to use link:#cluster-policy-and-local-policy[local policy] to control
+who has access to their link:../../dev_guide/projects.html[projects]. Note that
+authorization is a separate step from link:authentication.html[authentication],
+which is more about determining the identity of who is taking the action.
+
+Authorization is managed using:
+
+[cols="1,7"]
+|===
+
+|[[rules-def]]*Rules* |Sets of permitted link:#action[verbs] on a set of
+link:../core_objects/overview.html[resources]. For example, whether something
+can `create` pods.
+
+|[[roles-def]]*Roles* |Collections of rules.
+link:authentication.html#users-and-groups[Users and groups] can be associated
+with, or _bound_ to, multiple link:#roles[roles] at the same time.
+
+|[[bindings]]*Bindings* |Associations between users and/or groups with a
+link:#roles[role].
+
+|===
+
+Rules, roles, and bindings can be visualized
+link:../../admin_guide/manage_authorization_policy.html#viewing-roles-and-bindings[using
+the CLI]. For example, consider the following excerpt from
+link:../../admin_guide/manage_authorization_policy.html#viewing-roles-and-bindings[viewing
+a policy], showing rule sets for the *admin* and *basic-user*
+link:#roles[default roles]:
+
+====
+
+[options="nowrap"]
+----
+admin			Verbs					Resources															Resource Names	Extension
+			[create delete get list update watch]	[projects resourcegroup:exposedkube resourcegroup:exposedopenshift resourcegroup:granter secrets]				[]
+			[get list watch]			[resourcegroup:allkube resourcegroup:allkube-status resourcegroup:allopenshift-status resourcegroup:policy]			[]
+basic-user		Verbs					Resources															Resource Names	Extension
+			[get]					[users]																[~]
+			[list]					[projectrequests]														[]
+			[list]					[projects]															[]
+			[create]				[subjectaccessreviews]														[]		IsPersonalSubjectAccessReview
+----
+====
+
+The following excerpt from
+link:../../admin_guide/manage_authorization_policy.html#viewing-roles-and-bindings[viewing
+policy bindings] shows the above roles bound to various users and groups:
+
+====
+
+[options="nowrap"]
+----
+RoleBinding[admins]:
+				Role:	admin
+				Users:	[alice system:admin]
+				Groups:	[]
+RoleBinding[basic-user]:
+				Role:	basic-user
+				Users:	[joe]
+				Groups:	[devel]
+----
+====
+
+== Evaluating Authorization
+
+Several factors are combined to make the decision when OpenShift evaluates
+authorization:
+
+[cols="1,7"]
+|===
+
+|[[identity]]*Identity* |Both the user name and list of groups the user belongs to.
```

Comment:
`Identity` means something different in the authentication, authorization, admission access control chain.  Internally it's called `UserInfo`.  @liggitt I'd avoid the reuse, but I don't know what else a user might consider it.